.github/actions/dotnet-build/action.yaml

```diff
@@ -0,0 +1,144 @@
+name: Dotnet - Build
+description: Build a dotnet project
+
+inputs:
+  solution:
+    description: Is the path to a solution file.
+    required: true
+
+  project:
+    description: Is the path to a project file.
+    required: true
+
+  configuration:
+    description: Is the path to a configuration file.
+    required: true
+
+  migrations:
+    description: Is the path to a migrations file, if this project uses them.
+    required: false
+    default: ""
+
+  sdk-version:
+    description: Is a complete sdk version in format x.y.zzz
+    required: false
+    default: ""
+
+  runtime-version:
+    description: Is a complete runtime version in format x.y.z
+    required: false
+    default: ""
+
+runs:
+  using: composite
+  steps:
+    - name: Load - global.json
+      shell: bash
+      run: |
+        DIRECTORY=$(dirname '${{ inputs.solution }}')
+        test -f "$DIRECTORY/global.json" || exit 0
+        echo "DIRECTORY=$DIRECTORY" | tee -a $GITHUB_ENV
+        echo "SDK=$(jq -rc '.sdk.version' "$DIRECTORY/global.json")" | tee -a $GITHUB_ENV
+        echo "RUNTIME=$(jq -rc '.runtime.version' "$DIRECTORY/global.json")" | tee -a $GITHUB_ENV
+    - name: Overwrite - if given optional input
+      shell: bash
+      run: |
+        if [ ! -z "${{ inputs.sdk-version }}" ]; then
+          echo "SDK=${{ inputs.sdk-version }}" | tee -a $GITHUB_ENV
+        fi
+        if [ ! -z "${{ inputs.runtime-version }}" ]; then
+          echo "RUNTIME=${{ inputs.runtime-version }}" | tee -a $GITHUB_ENV
+        fi
+
+    - name: Fail - if versions are not present
+      shell: bash
+      run: |
+        set +e
+        ! test -z "${{ env.SDK }}"
+        ! test -z "${{ env.RUNTIME }}"
+
+    - name: Fetch Dockerfile
+      shell: bash
+      run: wget https://raw.githubusercontent.com/Energinet-DataHub/acorn-actions/adjust-dotnet-validations/Dockerfile.simplified -O Dockerfile # FIXME: this
```

Comment:
Should this be fetched from a branch in Acoun repo? Why not use the dockerfile in our own repo?

Reply:
The platform requires that we release our deployments as docker images. it requires that they follow specific practices. Ratatosk has agreed to handle the responsibility of the Dockerfile, because of that. That is a win in my book.